lib/std.rs

```diff
@@ -145,6 +145,62 @@ impl<T> From<std_Option<T>> for Option<T> {
     }
 }
 
+impl<A: record::FromRecord + serde::de::DeserializeOwned + Default> record::FromRecord
+    for std_Option<A>
+{
+    fn from_record(val: &record::Record) -> result::Result<Self, String> {
```

Comment:
it's in fact trickier, because the implementation has to know about this Record type. So then that has to be documented someplace.